dace/sdfg/state.py

```diff
@@ -803,13 +1164,14 @@ def __init__(self, label=None, sdfg=None, debuginfo=None, location=None):
             :param debuginfo: Source code locator for debugging.
         """
         from dace.sdfg.sdfg import SDFG  # Avoid import loop
+        OrderedMultiDiConnectorGraph.__init__(self)
+        ControlFlowBlock.__init__(self, label)
         super(SDFGState, self).__init__()
```

Comment:
there is a proper `super`  version in Python 3 that does this correctly, please adapt.

Reply:
Upon inspection, the alternative method seems to be inadequate here due to the mismatch in call signatures to the different superclasses' init functions. I am not an expert on multiple inheritance in Python though, so I may be missing something, too.